Specimen/Specimen-BoneMarrowAspiration-Example.json

```diff
@@ -0,0 +1,44 @@
+{
+  "resourceType": "Specimen",
+  "id": "Specimen-BoneMarrowAspiration-Example",
+  "identifier": [
+    {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
+      "value": "RR812434"
+    }
+  ],
+  "accessionIdentifier": {
+    "system": "https://www.leedsth.nhs.uk/path",
+    "value": "RR8123745"
```

Comment:
Could add assigner ODS code RR86 to this identifier

Reply:
Shortened the code to RR8, valid ODS code for Leeds Teaching Hospital